Content.Shared/Singularity/Events/SingularityLevelChangedEvent.cs

```diff
@@ -0,0 +1,38 @@
+using Content.Shared.Singularity.Components;
+using Content.Shared.Singularity.EntitySystems;
+
+namespace Content.Shared.Singularity.Events;
+
+/// <summary>
+/// An event raised whenever a singularity changes its level.
+/// </summary>
+public sealed class SingularityLevelChangedEvent : EntityEventArgs
+{
+    /// <summary>
+    /// The new level of the singularity.
+    /// </summary>
+    public readonly ulong NewValue;
+
+    /// <summary>
+    /// The previous level of the singularity.
+    /// </summary>
+    public readonly ulong OldValue;
```

Comment:
why do these need to be ulong isnt it like 1-6?

Reply:
Converted to byte